README.md

```diff
@@ -30,36 +31,50 @@ ______________________________________________________________________
 
 **Thunder makes PyTorch models Lightning fast.**
 
-Thunder is a source-to-source compiler for PyTorch. It makes PyTorch programs faster by combining and using different hardware executors at once (ie: nvFuser, torch.compile, cuDNN, and TransformerEngine FP8).
+Thunder is a source-to-source compiler for PyTorch. It makes PyTorch programs faster by combining and using different hardware executors at once (for instance, [nvFuser](https://github.com/NVIDIA/Fuser), [torch.compile](https://pytorch.org/docs/stable/torch.compiler.html), [cuDNN](https://developer.nvidia.com/cudnn), and [TransformerEngine FP8](https://github.com/NVIDIA/TransformerEngine)).
+
+It supports single accelerators (such as GPUs and TPUs) and also works in multi-GPU settings.
```

Comment:
Adding TPUs makes it sound like we have an XLA executor which we don't

Reply:
In that cade, why don't we just simplify "accelerators" to GPUs? More people would be familiar with that.

Reply:
Made some updates, let me know what you think.

Reply:
Looks good to me. Let me know if you'd like me to merge it or if you want Luca to check it out first

Reply:
If there's nothing wrong, I'd say let's merge. We can always make an additional pass later.